orderbook/src/api/get_orders.rs

```diff
@@ -29,25 +29,35 @@ struct Query {
     include_insufficient_balance: bool,
     #[serde(default)]
     include_unsupported_tokens: bool,
+    page_size: Option<u32>,
+    page: Option<u32>,
 }
 
 impl Query {
     fn order_filter(&self) -> Result<OrderFilter, &'static str> {
         if self.owner.is_none() && self.sell_token.is_none() && self.buy_token.is_none() {
             return Err("need to set at least one of owner, sell_token, buy_token");
         }
-        let to_h160 = |option: Option<&H160Wrapper>| option.map(|wrapper| wrapper.0);
-        Ok(OrderFilter {
-            min_valid_to: self.min_valid_to,
-            owner: to_h160(self.owner.as_ref()),
-            sell_token: to_h160(self.sell_token.as_ref()),
-            buy_token: to_h160(self.buy_token.as_ref()),
-            exclude_fully_executed: !self.include_fully_executed,
-            exclude_invalidated: !self.include_invalidated,
-            exclude_insufficient_balance: !self.include_insufficient_balance,
-            exclude_unsupported_tokens: !self.include_unsupported_tokens,
-            uid: None,
-        })
+
+        return match self.page_size {
+            Some(i) if !(1..=100).contains(&i) => Err("page_size must be between 1 and 100"),
```

Comment:
Is this creating an array in-memory array of 100 elements for each query just to test belonging of the page size to it in order to validate? 

Why not just having a `>=1` `<=100` conditions?

Reply:
Clippy complains :stu

Reply:
Yep, that's clippy :)
But to be honest, this way of declaring the interval looks more clear and concise to me. 

Also, the range is hard coded, I'd like to get a const for it

Reply:
ok, rust is too modern. I find it a bit weird, but im sure theres a reason, and i don't want to make clippy upset